### PR TITLE
Introducing `GrafanaEventAggragator` thread

### PR DESCRIPTION
**Pull request by fruch**
This thread checks every few minute for duplication of events in grafana
and delete some of the similar event in that time window.

this is to allow us to clearly see what's going on in the granfa more clearly.

NOTE: those events are not removed from any of the event log files

Done as part of: https://trello.com/c/Z0IKl933

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I gave variables/functions meaningful self-explanatory names
- [ ] I didn't leave commented-out/debugging code
- [ ] I didn't copy-paste code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: sdcm/cluster_aws.py [hunk 1]
```diff
@@ -420,7 +420,7 @@ def __init__(self, ec2_instance, ec2_service, credentials, parent_cluster,  # py
                           'user': ami_username,
                           'key_file': credentials.key_file}
         self._spot_aws_termination_task = None
-
+        self._wait_public_ip()
```
Comment (fruch) [inline sdcm/cluster_aws.py]: this should be in it's own PR

File: sdcm/utils/grafana.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    """
+    This is a background thread that scans grafana annotations in a `time_window` (in minutes)
+    find `max_duplicated events in that time windows, and delete all of the others similar events
+    """
+
+    def __init__(self, params):
+        self.stop_event = threading.Event()
+        self.url_set = threading.Event()
+        self.grafana_base_url = ''
+        # TODO: add those to sct_events.py
```
Comment (fruch) [inline sdcm/utils/grafana.py]: reminder to actually do this

Comment (fruch) [reply sdcm/utils/grafana.py]: take it our of sct_config.py

File: sdcm/sct_events.py [hunk 3]
```diff
@@ -538,72 +538,42 @@ def terminate(self):
 class GrafanaAnnotator(threading.Thread):
```
Comment (fruch) [inline sdcm/sct_events.py]: move to utils/grafana.py

File: sdcm/utils/grafana.py [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+        if params is None:
+            params = dict()
+        self.time_window = params.get('event_aggregator_time_window', 5)
```
Comment (bentsi) [inline sdcm/utils/grafana.py]: decrease to 60-90 sec

File: sdcm/sct_events.py (hunk 3 above)
Comment (bentsi) [inline sdcm/sct_events.py]: move to `utils/grafana.py`
